dspace-api/src/main/java/org/dspace/content/authority/service/ChoiceAuthorityService.java

```diff
@@ -168,4 +168,9 @@ public Choices getBestMatch(String fieldKey, String query, Collection collection
 
     public ChoiceAuthority getChoiceAuthorityByAuthorityName(String authorityName);
 
+    /**
+     * This method has been created to have a way of clearing the cache kept inside the service
+     */
+    public void clearController();
```

Comment:
A minor thing, but I'd recommend renaming this to be `clearCache();` as that aligns better with the method description here.  The word "Controller" is confusing to me, as this is not a Spring Controller (which is the first thing that comes to mind here).

Reply:
This has been renamed to clearCache